packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -278,6 +280,9 @@ class DayPicker extends StatelessWidget {
   /// Optional user supplied predicate function to customize selectable days.
   final SelectableDayPredicate selectableDayPredicate;
 
+  /// {@macro flutter.gestures.recognizer.dragStartBehavior}
+  final DragStartBehavior dragStartBehavior;
```

Comment:
i don't think that macro actually explains what this does in a meaningful way here, since there's no onStart to speak of here.

Reply:
the same applies to most of the places you use this macro, in fact

Reply:
Ok, that makes sense. What would better documentation for all these cases? I feel like the result from using start on all of these widgets produces slightly different results. Do I need to go through and detail exactly what that would be or would a more general "begins drag behavior upon winning the arena vs on down" be sufficient?

Reply:
For each one we should write specific documentation for what it means there. (See the style guide for writing prompts for documentation.) If there's commonalities, then it makes sense to use the macro feature to share the common text.

Reply:
So is the above comment I wrote not good for all of them? Since it's drag functionality for all of them, I don't think we need to specify super precisely what will change?

Reply:
The template says "Configure the behavior of offsets sent to [onStart].", but e.g. the date_picker doesn't have an onStart. It goes on to talk about things like the arena and so on, but someone using the date picker widget doesn't know anything about that. What they care about is what it means for the date picker itself, and the interactions that the date picker has. For example, for the YearPicker it would say something about how it controls whether when someone drags the list view with the years in it, the scrolling should follow the finger precisely, but jump at the start, or whether it should avoid any jumping behaviour, but have some lag at the start.

As you write this for each widget to which you added the dragStartBehavior, you may find some don't actually have any effect. For example, it's not clear to me that it does anything at all on DayPicker.